x-pack/plugin/inference/src/yamlRestTest/resources/rest-api-spec/test/inference/10_semantic_text_inference.yml

```diff
@@ -171,17 +167,14 @@ setup:
           index: test-sparse-index
           id: doc_1
 
-    - match:  { _source.inference_field: "inference test" }
-    - match:  { _source.another_inference_field: "another inference test" }
+    - match:  { _source.inference_field.raw: "inference test" }
+    - match:  { _source.inference_field.inference.chunks.0.text: "inference test" }
+    - match:  { _source.inference_field.inference.chunks.0.embeddings: $inference_field_embedding }
+    - match:  { _source.another_inference_field.raw: "another inference test" }
+    - match:  { _source.another_inference_field.inference.chunks.0.text: "another inference test" }
+    - match:  { _source.another_inference_field.inference.chunks.0.embeddings: $another_inference_field_embedding }
```

Comment:
I think this was an issue with the old version of the test as well, but since the `sparse-inference-id` inference service always returns the same embeddings, I don't think this actually tests that the embeddings were not recalculated.

Not sure how to fix this short-term, but maybe we flag it for follow-up?

Reply:
++ for a follow-up. We need more tests anyway ;)